pyproject.toml

```diff
@@ -49,7 +50,7 @@ tests = [
   "pytest-remotedata",
 ]
 [project.urls]
-'documentation' = 'https://asdf.readthedocs.io/en/stable'
+'documentation' = 'https://www.asdf-format.org/projects/asdf/en/latest'
```

Comment:
This is giving me a 404. What's causing the failure?

The old link was to 'stable' instead of 'latest'. I'm in favor of keeping 'stable' as it described the newest pypi release (whereas 'latest' describes the state of asdf main development).

Reply:
Once I add asdf as a subproject in RTD this url will work (the "old" url will still function). I will also change the default to stable